### PR TITLE
pagetree unit tests initial setup

### PR DESCRIPTION
**Pull request by vxsx**
disregard the fixture

### CODE REVIEW

File: cms/tests/frontend/unit/fixtures/pagetree.html [hunk 1]
```diff
@@ -0,0 +1,330 @@
+<div class="cms-pagetree cms-pagetree-container">
```
Comment (FinalAngel) [inline cms/tests/frontend/unit/fixtures/pagetree.html]: maybe some instructions on how to build that fixture might be helpful

Comment (vxsx) [inline cms/tests/frontend/unit/fixtures/pagetree.html]: fixture is not final, i'd wait a bit first :)

Comment (FinalAngel) [inline cms/tests/frontend/unit/fixtures/pagetree.html]: 👍

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**